src/v35.js

```diff
@@ -48,19 +23,23 @@ export default function (name, version, hashfunc) {
     }
 
     if (typeof namespace === 'string') {
-      namespace = uuidToBytes(namespace);
+      namespace = parse(namespace);
     }
 
-    if (!Array.isArray(value)) {
-      throw TypeError('value must be an array of bytes');
+    if (!value[Symbol.iterator]) {
```

Comment:
[Symbol.iterator doesn't seem to be supported in IE11](https://caniuse.com/#search=symbol.iterator).

However babel should take care of that for the browser builds.

Reply:
Removed Symbol.iterator check for the time being.